src/html/share.html

```diff
@@ -9,38 +9,6 @@
 </head>
 
 <body>
-    <div id='catblocks-code-container'>
+    <div id='catblocks-code-container'></div>
 </body>
-<script>
-    // share get exported by the bundle
-    share.init({
-        'container': 'catblocks-code-container',
-        'renderSize': 0.75,
-        'language': 'en',
-        'shareRoot': '/',
-        'media': 'media/',
-        'noImageFound': 'No_Image_Available.jpg',
-    });
-
-
-    // render my code.xml file
-    window.onload = function() {
-        share.parser.convertProgramUri(`assets/share/code.xml`)
-            .then(xmlDoc => {
-                console.log(xmlDoc);
-                const div = document.getElementById('catblocks-code-container');
-                share.injectAllScenes(div, xmlDoc, {
-                    object: {
-                        programRoot: 'assets/share/'
-                    }
-                });
-            })
-            .catch(err => {
-                console.error(`Failed to parse catroid file.`);
-                console.error(err);
-            });
-    };
-
-</script>
```

Comment:
moved javascript code in share.html and testing.html to index.js. So we need to call `setLocale()` only once.